rero_ils/modules/ebooks/utils.py

```diff
@@ -80,18 +80,17 @@ def create_document_holding(record):
     """Create a document and a holding for a harvested ebook."""
     harvested_sources = get_harvested_sources(record)
     new_record = None
-    doc_created = False
     for harvested_source in harvested_sources:
         if org := Organisation.get_record_by_online_harvested_source(
                 source=harvested_source['source']):
-            if not doc_created:
+            if not new_record:
```

Comment:
it seems `new_record` is always None at this moment. Maybe you can skip this  `if` statement.